CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Changelog
```

Comment:
I don't remember any particular thing for now, but would it make sense to add a few more notes to the changelog as this 6.0.0 brought other improvements and fixes? Not only the migration, but also other things that we did not bring to the OS in the past? Maybe go through out changelog and add the most important things here

Reply:
Also, can we create a release and/or tag to start separating this for people to easily see and use particular versios?